src/Articulate/BlogMlExporter.cs

```diff
@@ -20,7 +20,7 @@ public class BlogMlExporter
         public BlogMlExporter(UmbracoContext umbracoContext, IFileSystem fileSystem)
         {
             _umbracoContext = umbracoContext;
-            _applicationContext = _umbracoContext.Application;
+            _applicationContext = ApplicationContext.Current;
```

Comment:
This cannot happen, no singletons please, what was there before was much better and doesn't rely on singletons